src/fides/data/sample_project/.fides/fides.toml

```diff
@@ -0,0 +1,36 @@
+[database]
+server = "fides-db" # TODO: localhost?
```

Comment:
This gets a little bit complicated @NevilleS , as we actually need two separate configuration values here (this is also the same way that I dev locally). 

I often (almost always) run `nox -s dev` and then use the CLI with my local installation of fides. This works because `fides.toml` has the correct values for local development (everything is set to `localhost`) while in the container itself we override the values for docker-style domains.

I imagine things will work the same here. The docker application, internally, will use this sample `fides.toml` file to configure itself, but the local user (aka the person who just ran `fides deploy`) will then be able to run from their CLI using our pre-defined default values (`localhost`, etc.)

I'll make sure this is working as this is the path I'd like to go down